core/load/elasticsearch.py

```diff
@@ -122,15 +125,8 @@ def bulk_upsert_embeddings(
     ) -> None:
         index_name = target.index_name
         field_name = target.field_name
-        num_dimensions = len(embeddings[0])
         num_embeddings = len(embeddings)
 
-        if not all(len(embedding) == num_dimensions for embedding in embeddings):
```

Comment:
@rebasedming did you want to delete this or was it just deleted as part of trying to debug?

Everything else should be ready

Reply:
Yes this was intentional!